src/main/java/edu/tamu/scholars/middleware/discovery/component/solr/SolrIndexer.java

```diff
@@ -69,17 +69,17 @@ public void init() {
                 try {
                     SchemaRequest.AddField addFieldRequest = new SchemaRequest.AddField(fieldAttributes);
                     addFieldRequest.process(solrClient, collectionName);
-                } catch (Exception e) {
-                    logger.debug("Failed to add field", e);
-                }
 
-                if (fieldType.copyTo().length > 0) {
-                    try {
-                        SchemaRequest.AddCopyField addCopyFieldRequest = new SchemaRequest.AddCopyField(name, Arrays.asList(fieldType.copyTo()));
-                        addCopyFieldRequest.process(solrClient, collectionName);
-                    } catch (Exception e) {
-                        logger.debug("Failed to add copy field", e);
+                    if (fieldType.copyTo().length > 0) {
+                        try {
+                            SchemaRequest.AddCopyField addCopyFieldRequest = new SchemaRequest.AddCopyField(name, Arrays.asList(fieldType.copyTo()));
+                            addCopyFieldRequest.process(solrClient, collectionName);
+                        } catch (Exception e) {
+                            logger.error("Failed to add copy field", e);
```

Comment:
This behavior will succeed to create the field if the only failure is a copy field failure.

Should we fail the entire field if the copy field fails?
This really comes down to the design strategy when handling failures.
I am fine either way.

Reply:
No. We will reserve this for configurability of rolling back the creation of a field. Which is out of scope of the issue.